### PR TITLE
Update build.gradle with a comment explaining version

### PR DESCRIPTION
**Pull request by reidbaker**
Followup to https://github.com/flutter/packages/pull/3960/files/eda00fe0ee329433a013b63a1b7727f90f2d0de2#r1192563559 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/camera/camera_android_camerax/example/android/build.gradle [hunk 1]
```diff
@@ -1,4 +1,7 @@
 buildscript {
+    // This version should intentionally be a 1.7.* verson and lower than the
```
Comment (gmackall) [inline packages/camera/camera_android_camerax/example/android/build.gradle:2]: Maybe and "lower than the version of kotlin-bom..."? Some might get confused that this is a version of kotlin gradle plugin, and go look for a version of kotlin gradle plugin in `packages/camera/camera_android_camerax/android/build.gradle`, but not find one.

Comment (reidbaker) [inline packages/camera/camera_android_camerax/example/android/build.gradle:2]:
```suggestion
    // This version should intentionally be a 1.7.* version and lower than the 
```

File: packages/camera/camera_android_camerax/example/android/build.gradle [hunk 2]
```diff
@@ -1,4 +1,7 @@
 buildscript {
+    // This version should intentionally be a 1.7.* verson and lower than the 
+    // version defined in packages/camera/camera_android_camerax/android/build.gradle.
```
Comment (reidbaker) [inline packages/camera/camera_android_camerax/example/android/build.gradle:3]:
```suggestion
    // version of kotlin-bom defined in packages/camera/camera_android_camerax/android/build.gradle.
```

File: packages/camera/camera_android_camerax/example/android/build.gradle (hunk 1 above)
Comment (reidbaker) [reply packages/camera/camera_android_camerax/example/android/build.gradle]: Done

**Review by gmackall — APPROVED** [review]
LGTM!

**Review by camsim99 — APPROVED** [review]

**Comment by Hixie** [comment]
test-exempt: only affects comments

(bonus points available for teaching the bot about gradle comment syntax; it already knows about Dart comments being automatically test exempt)

**Comment by gmackall** [comment]
> test-exempt: only affects comments
> 
> (bonus points available for teaching the bot about gradle comment syntax; it already knows about Dart comments being automatically test exempt)

I was curious how the parsing worked so I took a quick look and it looks like it should be as simple as adding gradle to the [list of recognized extensions](https://github.com/flutter/cocoon/blob/259a391efea8aa95d36b7fe9fad766ee42ee65ae/app_dart/lib/src/request_handlers/github/webhook_subscription.dart#L664), because the single line comment syntax [is the same](https://docs.groovy-lang.org/next/html/documentation/#_comments) as java/the others we are parsing (and single line comment are all we seem to support anyways).

**Pull request merged**